staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -562,6 +566,50 @@ func TestApfCancelWaitRequest(t *testing.T) {
 	})
 }
 
+type fakeFilterForReqDigest struct {
```

Comment:
nit: the convention is to avoid shortcuts; so probably should be:

fakeFilterForRequestDigest (or fakeFilterForDigest or sth)